ios/Podfile.lock

```diff
@@ -42,7 +42,7 @@ PODS:
     - FirebaseAnalytics (~> 5.1)
     - FirebaseAnalyticsInterop (~> 1.0)
     - FirebaseCore (~> 5.2)
-  - FirebaseInstanceID (3.7.0):
+  - FirebaseInstanceID (3.8.1):
```

Comment:
Why are all these updated? Did you run `pod update` ? Instead, please do `pod install --repo-update` which shouldn't update unrelated dependencies.